app/controllers/wheels/prizes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PrizesController < ApplicationController
```

Comment:
Have a look at how to namespace controllers, as I suggested. This and the `configurations` one should have a different naming, now that they're namespaced.